example/index.html

```diff
@@ -7,71 +7,97 @@
 <meta name="author" content="" />
 <meta name="viewport" content="width=device-width, initial-scale=1" />
 <meta name="robots" content="noindex, nofollow" />
+
 <style>
-	button {
-			width: 100px;
-			height: 44px;
-	}
+    button {
+        width: 120px;
+        height: 44px;
+    }
 </style>
-
 </head>
 <body>
   <h1>Notify.js demo</h1>
   <p>Notify.js is a simple wrapper for the Web Notification API. The GitHub repo is <a href="https://github.com/alexgibson/notify.js">here</a>.</p>
-  <p>Click the button below to fire up a notification. Currently supported in Chrome, Safari, Firefox and Firefox OS.</p>
-  <button id="my-button">Notify Me!</button>
+  <p>Click the buttons below to fire up a notification. Currently supported in Chrome, Safari, Firefox and Firefox OS.</p>
+  <div id="buttons">
+      <button id="btn-api">Notify Me!</button>
+      <button id="btn-service-worker">Notify Me from Service Worker!</button>
+  </div>
 
   <!-- JavaScript assets -->
   <script src="../dist/notify.js"></script>
   <script>
-    document.getElementById('my-button').addEventListener('click', function () {
+      var btnApi = document.getElementById('btn-api');
+      var btnServiceWorker = document.getElementById('btn-service-worker');
 
-        function onShowNotification () {
-            console.log('notification is shown!');
-        }
+      btnApi.addEventListener('click', function() {
+          doNotification(new Notify('Yo dawg!', {
+              body: 'This is an awesome notification using a service worker',
+              tag: 'My unique id',
+              notifyShow: onShowNotification,
+              notifyClose: onCloseNotification,
+              notifyClick: onClickNotification,
+              notifyError: onErrorNotification,
```

Comment:
But the problem is you're removing the working example that 99% of people are using today. Not everyone is up to implementing service workers just yet. Please reinstate it, thanks,